tests/tests_change_settings.yml

```diff
@@ -0,0 +1,114 @@
+- hosts: all
+  tasks:
+    - name: Ensure required packages are installed
+      package:
+        name: tuned
+        state: present
+
+    - name: Ensure kernel settings profile directory exists
+      file:
+        path: /etc/tuned/kernel_settings
+        state: directory
+        mode: 0755
+
+    - name: Generate a configuration for kernel settings
+      copy:
+        src: tuned/etc/tuned/basic_settings/tuned.conf
```

Comment:
You can't use this file anymore.  You'll need to use or create a file that doesn't have the bootloader cmdline settings in it.

Reply:
Hello. Thank you. Which file is it?

Reply:
> Hello. Thank you. Which file is it?

There is no such file that is the same as `tuned/etc/tuned/basic_settings/tuned.conf` but without the bootloader cmdline settings.  I suggest you create a new profile + file e.g. `tests/tuned/etc/tuned/simple_settings/tuned.conf` which is the same as `tests/tuned/etc/tuned/basic_settings/tuned.conf` but without the bootloader cmdline settings, and use that file in your test.

Reply:
Many, many thanks. Should be solved now.